### ISSUE TITLE
Feature request: add browse tribes on the expanded hamburger menu

### ISSUE THREAD

**Issue by pedromvpg** [stakwork/sphinx-relay#104]
Would be nice to have a way to access the tribes index from the app. Even if just a embedded browser, but something that allows the user to find more communities without leaving the app.

**Comment by pitoi** [comment]
Agreed, we'll look at a way to do this. We left it off of the main menu since we want to decouple Sphinx the app from the services so multiple people could run tribes servers and hosting nodes. That way none of this is centralized around what we're doing. It's a balance between ease of use and recreating the centralized problem we have now.
